docs/build.rst

```diff
@@ -3,45 +3,90 @@
 Installation
 ============
 
-Pre-configured installation (recommended)
------------------------------------------
+It is highly recommended that you use a pre-built version of h5py, either from a
+Python Distribution, an OS-specific package manager, or a pre-built wheel from
+PyPI.
 
-It's strongly recommended that you use a Python distribution or package
-manager to install h5py along with its compiled dependencies.  Here are some
-which are popular in the Python community:
+Be aware however that most pre-built versions lack MPI support, and that they
+are built against a specific version of HDF5. If you require MPI support, or
+newer HDF5 features, you will need to build from source.
 
-* `Anaconda <http://continuum.io/downloads>`_ or `Miniconda <http://conda.pydata.org/miniconda.html>`_ (Mac, Windows, Linux)
-* `Enthought Canopy <https://www.enthought.com/products/canopy/>`_ (Mac, Windows, Linux)
-* `PythonXY <https://code.google.com/p/pythonxy/>`_ (Windows)
+After installing h5py, you should run the tests to be sure that everything was
+installed correctly. This can be done in the python interpreter via::
 
-::
+    import h5py
+    h5py.run_tests()
 
-    conda install h5py  # Anaconda/Miniconda
-    enpkg h5py          # Canopy
+On Python 2.6, unittest2 must be installed to run the tests.
 
-Or, use your package manager:
+Pre-built installation (recommended)
+-----------------------------------------
 
-* apt-get (Linux/Debian, including Ubuntu)
-* yum (Linux/Red Hat, including Fedora and CentOS)
-* Homebrew (OS X)
-* pacman (Arch linux)
+Pre-build h5py can be installed via many Python Distributions, OS-specific
+package managers, or via h5py wheels.
 
+Python Distributions
+....................
+If you do not already use a Python Distribution, we recommend either
+`Anaconda <http://continuum.io/downloads>`_/`Miniconda <http://conda.pydata.org/miniconda.html>`_
+or
+`Enthought Canopy <https://www.enthought.com/products/canopy/>`_, both of which
+support most versions of Microsoft Windows, OSX/MacOS, and a variety of Linux
+Distributions. Installation of h5py can be done on the command line via::
 
-.. _source_install:
+    $ conda install h5py
 
-Source installation on Linux and OS X
--------------------------------------
+for Anaconda/MiniConda, and via::
 
-You need, via apt-get, yum or Homebrew:
+    $ enpkg h5py
 
-* Python 2.6, 2.7, 3.3, 3.4, or 3.5 with development headers (``python-dev`` or similar)
-* HDF5 1.8.4 or newer, shared library version with development headers (``libhdf5-dev`` or similar)
-* NumPy 1.6.1 or later
+for Canopy.
 
-::
+Wheels
+......
+If you have an existing Python installation (e.g. a python.org download,
+or one that comes with your OS), then on Windows, MacOS/OSX, and
+Linux on Intel computers, pre-built h5py wheels can be installed via pip from
+PyPI::
 
     $ pip install h5py
 
+Additionally, for Windows users, `Chris Gohlke provides third-party wheels
```

Comment:
Cristoph builds the windows wheels we have on pypi for us.  I wonder if those are complied against mkl.